src/administration/horizontal_scaling.md

```diff
@@ -58,16 +58,36 @@ There are a few ways to work around this issue:
 
 Lemmy_server can be horizontally scaled, with a few caveats.
 
+Here's a quick example on how you could start 3 web servers, 3 federation servers and one scheduled task process:
+
+```
+lemmy_server --http-server=false --federate-activities=false # scheduled tasks
+lemmy_server --http-server=true --federate-activities=false --disable-scheduled-task # http server 1  
+lemmy_server --http-server=true --federate-activities=false --disable-scheduled-task # http server 2
+lemmy_server --http-server=true --federate-activities=false --disable-scheduled-task # http server 3
+
+# federation server 1/3
+lemmy_server --http-server=false --federate-activities=true --federate-process-index=1 --federate-process-count=1 --disable-scheduled-tasks
+# federation server 2/3
+lemmy_server --http-server=false --federate-activities=true --federate-process-index=1 --federate-process-count=2 --disable-scheduled-tasks
+# federation server 3/3
+lemmy_server --http-server=false --federate-activities=true --federate-process-index=1 --federate-process-count=3 --disable-scheduled-tasks
```

Comment:
I think you confused federate-process-index and federate-process-count here. The first should increment from 1 to 3 and the latter always be 3.

Reply:
Bump. Also CI is failing on formatting.

Reply:
good point. fixed